docs/releases/2.0.txt

```diff
@@ -190,6 +190,10 @@ Models
 * The new :class:`~django.db.models.functions.StrIndex` database function
   finds the starting index of a string inside another string.
 
+* Added the ``chunk_size`` parameter to :meth:`.QuerySet.iterator` to control
```

Comment:
To follow the style of the previous note:

The new ``chunk_size`` parameter of :meth:`.QuerySet.iterator` controls...